webapp/components/wpt-colors.html

```diff
@@ -0,0 +1,58 @@
+<link rel="import" href="../bower_components/paper-styles/color.html">
+
+<dom-module id="wpt-colors">
+  <template>
+    <style>
+      .passes-none { background-color: var(--paper-red-400); }
+      .passes-hardly { background-color: var(--paper-orange-400); }
+      .passes-a-few { background-color: var(--paper-amber-400); }
+      .passes-half { background-color: var(--paper-yellow-400); }
+      .passes-lots { background-color: var(--paper-lime-400); }
+      .passes-most { background-color: var(--paper-light-green-400); }
+      .passes-all { background-color: var(--paper-green-400); }
+    </style>
+  </template>
+
+  <script>
+    const wpt = window.wpt || {};
+    wpt.colors = [
+      { class: 'passes-none', colorVar: '--paper-red-400', rgb: [239, 83, 80] },
```

Comment:
Can you link to the canonical definition of these that these numbers should match?

Reply:
Linked above.